apis/fluentbit/v1alpha2/clusterinput_types_test.go

```diff
@@ -26,6 +27,14 @@ var inputExpected = `[Input]
     Tag    logs.foo.bar
     Rate    3
     Samples    5
+[Input]
+    Name    prometheus_scrape
```

Comment:
`prometheus_scrape_metrics` is ok instead of `prometheus_scrape`

Reply:
![image](https://user-images.githubusercontent.com/24885903/186301238-bc66dbea-72bd-4494-aeec-7c3c8e57cbdf.png)

![image](https://user-images.githubusercontent.com/24885903/186301552-f5327cdd-23be-46e2-9542-0d642e30fa2e.png)

It seems that it must be the "prometheus_scrape" field

Reply:
What's the content of the configuration file you used?
Does the name of the plugin in the configuration file is set to `prometheus_scape`?

Reply:
![image](https://user-images.githubusercontent.com/24885903/186304893-1465e1bb-256a-42d9-86b8-d9e4007ab43c.png)
![image](https://user-images.githubusercontent.com/24885903/186305189-e7b5ae67-c222-4733-b171-575feebe519a.png)

"prometheus_scrape_metrics" is not effective

![image](https://user-images.githubusercontent.com/24885903/186306552-70e066da-cd74-482f-a8c1-76773fecfe05.png)
![image](https://user-images.githubusercontent.com/24885903/186306704-f2cf0d95-2885-4eb7-adcd-08ec1d32e0ff.png)

"prometheus_scrape"  is  ok

Reply:
ok, let's keep it this way